app/controllers/price_groups_controller.rb

```diff
@@ -28,7 +28,7 @@ def show
 
   # GET /facilities/:facility_id/price_groups/:id/users
   def users
-    unless @price_group_ability.can?(:read, UserPriceGroupMember)
+    unless SettingsHelper.feature_on?(:user_based_price_groups) && @price_group_ability.can?(:read, UserPriceGroupMember)
```

Comment:
Do we have any specs that hit this controller action that will now fail if the feature is off?